frontend/src/diagram/DiagramWebSocketContainer.tsx

```diff
@@ -481,12 +481,12 @@ export const DiagramWebSocketContainer = ({
   );
 
   const resizeElement = useCallback(
-    (diagramElementId, newPositionX, newPositionY, newWidth, newHeight) => {
+    (nodeId, newPositionX, newPositionY, newWidth, newHeight) => {
```

Comment:
The behavior of the frontend should stay the same, this change is related to an internal API of the backend